components/authz-service/storage/postgres/migration/legacy/iam.go

```diff
@@ -428,6 +434,11 @@ func convertV1Resource(resource string) (string, error) {
 		return "*", nil
 	}
 
+	singleTermsToMigrate := []string{"nodes", "events", "license", "nodemanagers", "service_groups"}
+	if len(terms) == 1 && !includes(singleTermsToMigrate, terms[0]) {
+		return "", nil
+	}
+
 	switch terms[0] {
 	case "service_info":
 		switch terms[1] {
```

Comment:
Did we test all these switch cases that use `terms[1]` with no guard as well?

Reply:
Yes, all single term resource policies are now tested [in this test](https://github.com/chef/automate/pull/3351/files/833641eded1ff2bd358ad97fcf3577325128ae85#diff-b05308e7d958693621e9a536ed26554dR234).

Lines 438-441 provide the guard for any single-term resources. We only let through resources like `nodes` and `license`, which are used in v1 policy annotations.